api/tests/test_cloud_record.py

```diff
@@ -170,6 +102,57 @@ def _saved_messages(self, message_path):
         """Return a list of messages under message_path."""
         return glob.glob(message_path)
 
+    def _make_record_post(self, message, expected_status,
```

Comment:
I feel this should be called something like `_check_record_post` as the asserts are really the key bit.